### DISCUSSION TITLE
Navigation drawer direction not found

### DISCUSSION THREAD

**Discussion by sharad740** [flet-dev/flet#3473] [Ideas]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

```
page.drawer = ft.NavigationDrawer(
        
        indicator_shape=ft.BeveledRectangleBorder(radius=1),
        controls=[
            ft.NavigationDrawerDestination(selected_icon=ft.icons.LOGIN_OUTLINED, icon_content=ft.Icon(ft.icons.LOGIN_OUTLINED), label="Login"),
            ft.NavigationDrawerDestination(selected_icon=ft.icons.LOGOUT_OUTLINED, icon_content=ft.Icon(ft.icons.LOGOUT_OUTLINED), label="Log Out"),
        ]
    )
    
    page.bottom_appbar = ft.BottomAppBar(
        bgcolor=ft.colors.WHITE,
        disabled=False,
        content=ft.Row(
            controls=[
                ft.IconButton(icon=ft.icons.HOME_OUTLINED, icon_color=ft.colors.BLACK87),
                ft.IconButton(icon=ft.icons.CHAT_BUBBLE_OUTLINE_OUTLINED, icon_color=ft.colors.BLACK87),
                ft.IconButton(icon=ft.icons.ADD_CIRCLE_OUTLINE_ROUNDED, icon_color=ft.colors.BLACK87),
                ft.IconButton(icon=ft.icons.NOTIFICATIONS_OUTLINED, icon_color=ft.colors.BLACK87),
                ft.IconButton(icon=ft.icons.PERSON_OUTLINED, icon_color=ft.colors.BLACK87,on_click=show_person_drawer),
            ],
            alignment=ft.MainAxisAlignment.SPACE_BETWEEN,  # Distribute items evenly
            vertical_alignment=ft.MainAxisAlignment.CENTER,  # Vertical alignment if needed
        ),
    )
```

Notification drawer direction not found !

**Comment by ndonkoHenri** [comment]
Please be more precise with your issue. I can't understand what you mean.
Also, when sharing code, make sure it is directly runnable.

**Comment by sharad740** [comment]
There is direction parms in most of frontend lib for drawers to set direction of the OffCanvas. I am requesting for feature request on Widgets of Drawer.

**Comment by ndonkoHenri** [comment]
Can you share an example of what you mean?

**Comment by sharad740** [comment]
https://mui.com/material-ui/react-drawer/#anchor

As flet heavily do frontend, Drawer having directions as parms in a widget would ease the flow and speed of development.

**Comment by ndonkoHenri** [comment]
Atm Flet supports two types (left and right) of drawers: https://flet.dev/docs/controls/navigationdrawer
I don't think Flutter (the framework on which Flet is based) supports top and bottom drawers.

**Reply by sharad740** [reply]
This component would fulfill my requirement at https://flet.dev/docs/controls/bottomsheet. Thank you @ndonkoHenri  ..
